### PR TITLE
3.0-ify toolbox/app-storage

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

**Comment by e111077** [comment]
Currently app storage elements are broken when it comes to offline functionality. I'll take a deeper dive tomorrow

**Comment by e111077** [comment]
App storage is blocked on tools supporting import.meta on other browsers

**Comment by e111077** [comment]
I have a fix for app-storage. Should come out in our next elements release

**Comment by katejeffreys** [comment]
\o/ reverted

File: app/3.0/toolbox/app-storage.md [hunk 1]
```diff
@@ -4,16 +4,12 @@ title: App storage
...
-<div>
-{% include 'outdated.html' %}
-</div>
-
 The app storage family of elements gives you a new set of tools for managing data in your app. The
 initial set includes ready-made elements for integrating with Firebase and PouchDB.
 
 ## Firebase integration
 
-The Firebase 3.0.0 SDK supports a new set of Firebase elements, built with app storage, called
```
Comment (arthurevans) [inline app/3.0/toolbox/app-storage.md]: I think we're going to need to remove this section (or add a note that the PolymerFire elements haven't been updated to 3.0 yet).

Comment (arthurevans) [reply app/3.0/toolbox/app-storage.md]: Talked to @e111077, yes, let's cut this section for now. These elements are outside our org so we'll need to coordinate.

**Comment by katejeffreys** [comment]
Removed section on PolymerFire. Are the other references to mirroring Firebase dbs still good?

**Comment by arthurevans** [comment]
Those comments are fine. LGTM!

**Pull request merged**